Strategic-Initiatives.md

```diff
@@ -0,0 +1,39 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  The goal of the TSC is to have a `champion` for each of these
+initiatives and to support the initiatives in order to enable their
+success.
+
+A review of the initiatives will be a standing item on the TSC agenda (even
+if the update is `nothing new`) as a way to ensure they are active
+and have the support needed.
+
+# Current Intiatives
+
+| Initiative        | Champion        | Links                                                            |
+|-------------------|-----------------|------------------------------------------------------------------|
+| http2             | James Snell     |                                                                  |
+| Modules           | Myles Borins    | https://github.com/nodejs/node-eps/blob/master/002-es-modules.md |
+| N-API             | Michael Dawson  | https://github.com/nodejs/abi-stable-node                        |
+| OpenSSL Evolution | Rod Vagg        | https://github.com/nodejs/TSC/issues/364                         |
+| Workers           | Anna Henningson | https://github.com/nodejs/worker                                 |
+| Error Messages    | James Snell     | https://github.com/nodejs/node/issues/11273i                     |
+| CVE Management    | Michael Dawson  | https://github.com/nodejs/security-wg/issues/33                  |
+| Core Promise APIs | James Snell     |                                                                  | 
+| TSC Governance    | Myles Borins    | https://github.com/nodejs/TSC/issues/383                         |
+| New Streams APIs  | James Snell     | https://github.com/ nodejs/node#16414                            | 
```

Comment:
A nit: `https://github.com/ nodejs/node#16414` -> `https://github.com/nodejs/node/pull/16414` ?

Reply:
done

Reply:
Space is fixed, but the URL is still wrong?